pkg/client/clientset/versioned/clientset.go

```diff
@@ -57,7 +57,7 @@ func NewForConfig(c *rest.Config) (*Clientset, error) {
 	configShallowCopy := *c
```

Comment:
We need to remove `clientset` in the future once we finish implementation.

Reply:
This need more discussion. Do we want this repo to host all client sets or a separate repo? 

We are not sure which project interact with training crds. If there's any, we should prioritize the story

Reply:
Sounds good, let's track it somewhere.

Reply:
I create an issue here https://github.com/kubeflow/tf-operator/issues/1332 to track this discussion